Homework_6/commentify/frontend/src/components/OnePost/OnePost.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import Comment from "../Comment/Comment";
+import classes from "./OnePost.module.css";
+import upSort from "../../assets/upsort.png";
+import downSort from "../../assets/downsort.png";
+import { ASCENDING, DESCENDING } from "../../constants/text";
+import sort from "../../helpers/sort";
+import AddComment from "../AddComment/AddComment";
+import { deleteComment } from "../../Api/api";
+
+class OnePost extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sortDir: DESCENDING,
+      comments: sort(props.post.comments, DESCENDING, "rating"),
+    };
+  }
+
+  refreshComments = (updatedComment) => {
+    this.setState({
+      comments: sort(
+        [...this.state.comments, updatedComment],
+        this.state.sortDir,
+        "rating"
+      ),
+    });
+  };
+
+  deleteCom = async (postId, commentId) => {
+    // const updatedComments = this.state.comments.filter((com) => com.id !== id);
+    const updatedComments = await deleteComment(postId, commentId);
+    this.setState({ comments: updatedComments.data });
+  };
+
+  drawSortDirection = () => {
```

Comment:
It is always better to write like this Andranik jan; cutting JSX and create small from it part

Reply:
didn't understand. Did you mean it's good practice to create functions which returns part of components returning jsx?